G/GDAL/build_tarballs.jl

```diff
@@ -3,27 +3,28 @@
 using BinaryBuilder, Pkg
 
 name = "GDAL"
-upstream_version = v"3.6.2"
-version_offset = v"1.0.0"
+upstream_version = v"3.7.1"
+version_offset = v"0.0.0"
 version = VersionNumber(upstream_version.major * 100 + version_offset.major,
                         upstream_version.minor * 100 + version_offset.minor,
                         upstream_version.patch * 100 + version_offset.patch)
 
 # Collection of sources required to build GDAL
 sources = [
-    ArchiveSource("https://github.com/OSGeo/gdal/releases/download/v$upstream_version/gdal-$upstream_version.tar.gz",
-        "cd5bf004af85b88414676b0d5694d3192414e556961ee3d47164f3298078a818"),
+    GitSource("https://github.com/OSGeo/gdal.git",
```

Comment:
Just wondering, what's the reason for moving from ArchiveSource to GitSource?

Reply:
Github based ArchiveSource's are deprecated

Reply:
(as not reproducible)